packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/ios/gettingatoken.md

```diff
@@ -1,32 +1,50 @@
-This example shows part of the implementation of the `IDXClientDelegate` protocol functions the SDK calls for each response and for receiving the access token.
+Getting the authorization token takes two-steps. First, request a the token after the sign-in is successful. Then receive that token from the server. The access token is represented by a `Token` object from the `AuthFoundation` library that is a dependency for the Swift IDX SDK. You can store the token securely in the Keychain using `Credential`, another class from the `AuthFoundation` library.
 
-```swift
-
-import OktaIdx
-
-class SignInController {
-    ...
-
-    var currentResponse: Response? = nil
+This example shows part of the implementation of the `InteractionCodeFlowDelegate` protocol functions that are called as part of getting the authorization token.
```

Comment:
We call it an access token in line 1, but now it's an authorization token. Can we call it "access token" again, or call out that the two are different please?

Reply:
I rewrote these two paras as they're a bit confusing because they were mixing older and newer terminology and ways of doing things. Let me know if this is OK:

> Request the access token after a successful sign-in. The access token is represented by a `Token` object from the `AuthFoundation` library that is a dependency for the Swift IDX SDK. You can store the token securely in the Keychain using `Credential`, another class from the `AuthFoundation` library.
>
> This example shows part of the implementation of the `InteractionCodeFlowDelegate` protocol functions that are called as part of getting the access token.